cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -58,17 +55,15 @@ def get_custom_report(appliance, custom_report_values):
 
 
 @pytest.mark.rhel_testing
-@pytest.mark.sauce
-@pytest.mark.tier(3)
 @pytest.mark.meta(blockers=[BZ(1531600, forced_streams=["5.9"])])
-@test_requirements.report
```

Comment:
You can remove this BZ as its [clone for 5.9 ](https://bugzilla.redhat.com/show_bug.cgi?id=1532654) is verified.